amplify/backend/function/userProfileDBHandlerDev/src/index.py

```diff
@@ -0,0 +1,79 @@
+import json
+import gremlin_python
+from gremlin_python.driver import client, serializer
+
+def lambda_handler(event, context):
+
+    try:
+        method = event["action"]
+    except:
+        return {
+        "statusCode": 78,
+        "body": "Method not found in request object."
+        }
+
+    # Neptune configuration
+    neptune_endpoint = "user-profiles-dev.cluster-cdbuvb1lszp1.us-west-2.neptune.amazonaws.com"
+    neptune_port = 8182
+
+    # Connect to Neptune
+    db = client.Client(f"wss://{neptune_endpoint}:{neptune_port}/gremlin", "g")
+
+    # DB rest functions
+    def db_get_user(username):
+        query = f"g.V().hasLabel('user').has('username', '{username}').values();"
+        return db.submit(query).all().result()
+
+    def db_create_user(userData):
+        firstName = userData["firstName"]
+        lastName = userData["lastName"]
+        dob = userData["DOB"]
+        username = userData["username"]
+        query = f"g.addV('user').property('firstName', '{firstName}').property('lastName', '{lastName}').property('DOB', '{dob}').property('username', '{username}');"
+        
+        db.submit(query)
+            
+        return "Query excecuted successfully"
```

Comment:
```suggestion
        db.submit(query)
        return "Query excecuted successfully"
```
Wrap this in a `try... except` statement.